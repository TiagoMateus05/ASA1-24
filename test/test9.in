7 25
2 1 3 3 6 1 4
7 1 3 4 2 7 3
2 4 1 6 3 4 6
3 2 5 1 1 2 1
1 7 3 4 1 2 1
7 4 4 1 2 3 5
4 3 7 2 7 2 7
3 2 1 4 1 7 2 4 4 1 7 1 1 3 1 3 7 6 7 3 3 5 7 6 2
5