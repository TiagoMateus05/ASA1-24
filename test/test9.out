1
((((((((((((((((((3 (2 1)) (4 1)) 7) 2) 4) 4) 1) 7) 1) 1) 3) 1) 3) 7) 6) (7 3)) 3) ((5 (7 6)) 2))
